### DISCUSSION TITLE
After changing the "Check Interval" value, how long does it take to use the new setting?

### DISCUSSION THREAD

**Discussion by GGitR** [Tzahi12345/YoutubeDL-Material#959] [Q&A]
In YouTube, I added a couple of videos to a YouTube playlist that I want to download immediately, so I changed the "Check Interval" setting from 36,000 to 60 and clicked "Save." I waited a couple of minutes, but the newly added videos did not start downloading, so I restarted the container (on my Synology NAS), and it seemed to take, as the videos started downloading. Is this the recommended way to do this? Or is there a better way?

**Comment by Tzahi12345** [comment]
Will be fixed with #1032!
